spec/features/admin_order_view_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+describe 'users past order views', type: :feature do
+  let(:admin) { User.create(username: 'admin', password: 'password',
+    password_confirmation: 'password', first_name: 'Justin',
+    last_name: 'Bieber', email: 'asdf@gmail.com',
+    role: 1) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 7, 14 should be on its own line.